### PR TITLE
Add users_create and users_index

### PR DESCRIPTION
**Pull request by jsteiner**

### CODE REVIEW

File: test/unit/cog_api/fake/users_test.exs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  use CogApi.FakeCase
+
+  alias CogApi.Fake.Users
+
+  doctest CogApi.Fake.Users
+
+  describe "index" do
+    it "returns a list of users" do
+      params = %{
+        first_name: "Leo",
```
Comment (drapergeek) [inline test/unit/cog_api/fake/users_test.exs]: :heart:

File: test/unit/cog_api/fake/users_test.exs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        last_name: "McGary",
+        email_address: "cos@example.com",
+        username: "chief_of_staff",
+        password: "supersecret",
+      }
+      {:ok, user} = Users.create(fake_endpoint, params)
+
+      assert present user.id
+      assert user.first_name == params.first_name
```
Comment (drapergeek) [inline test/unit/cog_api/fake/users_test.exs]: Should we be asserting all the other attributes here? This is our only check that it isn't just ignoring those other options.

**Comment by drapergeek** [comment]
LGTM

File: test/unit/cog_api/fake/users_test.exs (hunk 2 above)
Comment (jsteiner) [inline test/unit/cog_api/fake/users_test.exs]: Yes. Good catch.

**Pull request merged**